docs/Makefile

```diff
@@ -1,153 +0,0 @@
-# Makefile for Sphinx documentation
```

Comment:
I think eventually we're still planning on building sphinx docs. Although I am fine with this being deleted for now and bought back when we have sphinx docs to build.